packages/story-editor/src/app/story/effects/useHistoryEntry.js

```diff
@@ -27,8 +27,13 @@ import { useHistory } from '../../history';
 import deleteNestedKeys from '../utils/deleteNestedKeys';
 
 // Changes to these properties of elements do not create a new history entry.
-// Props to be added in
-const ELEMENT_PROPS_TO_IGNORE = [];
+const ELEMENT_PROPS_TO_IGNORE = [
+  'resource.baseColor',
```

Comment:
```suggestion
  'resource.baseColor',
  'resource.blurHash',
```

Can we add blurhash as well?

Reply:
Isn't that only added for media library items? So I don't think it's actually relevant here.

Reply:
It seems like they are added to story data. 
<img width="441" alt="Screenshot 2022-01-19 at 11 40 49" src="https://user-images.githubusercontent.com/237508/150123152-373fe2a0-7408-4363-90fe-47a231ec2d4b.png">

Reply:
> It seems like they are added to story data.

Since it's never needed inside a story (if I understand correctly?), should we remove it from adding to the story data instead? Or alternatively, should the blurhash be used instead of the `baseColor`?

Reply:
> It seems like they are added to story data.

Yes, but the blurhash generation is done _before_ an image is inserted to the page.

I don't think this scenario ever happens:

1. Add image to the page
2. Blurhash is added to the image
3. History is polluted

> Since it's never needed inside a story (if I understand correctly?), should we remove it from adding to the story data instead?

It doesn't hurt if it stays there, no?

Reply:
> It doesn't hurt if it stays there, no?

No, not really. Was just thinking in case it pollutes the history. If it doesn't then it's ok.

Reply:
No, that isn't the case. If you drag an image into canvas, you generate a local resource while while it is uploading. That is then replaces with real src etc. Then once loaded, blurhash and base color are backfilled. Meaning the resource woud be updated, polluting history.

Reply:
Does it hurt anything to ignore it?

Reply:
I thought the backfilling only updates resources in the library, _not_ the page. But I can now see that it does both:

https://github.com/google/web-stories-wp/blob/37cd92a734635bd1cbb8b878f2b3df7db35d4dd9/packages/story-editor/src/app/media/utils/useDetectBlurhash.js#L58-L62

So, alright, let's add it to the list.

Reply:
> Does it hurt anything to ignore it?

Nope, I can definitely add it here if it pollutes history.